src/components/Typography/index.tsx

```diff
@@ -217,7 +217,8 @@ type TypographyProps = {
   onMouseEnter?: (event: MouseEvent) => void
   tooltipProps?: Partial<TooltipProps>
   variant?: TypographyVariant
-} & XStyledProps
+} & XStyledProps &
+  React.HTMLAttributes<HTMLElement>
```

Comment:
Can we destructure in React import so we can directly use `HTMLAttributes` instead of `React.HTMLAttributes`